coderd/database/migrations/000047_workspace_resource_hide.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE workspace_resources
+    ADD COLUMN hide boolean NOT NULL;
```

Comment:
Whenever you add a new column it needs to have a defualt value, like `DEFAULT 'f'`.

Reply:
100%

Reply:
Fixed